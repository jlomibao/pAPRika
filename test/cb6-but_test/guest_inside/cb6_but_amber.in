PBC MD Simulation
 &cntrl
  imin =          0,
  ntx =           1,
  irest =         0,
  maxcyc =        0,
  ncyc =          0,
  dt =            0.002,
  nstlim =        1000,
  ntpr =          250,
  ntwe =          500,
  ntwr =          5000,
  ntwx =          250,
  ntxo =          1,
  ioutfm =        1,
  ntf =           2,
  ntc =           2,
  cut =           999.0,
  igb =           1,
  tempi =         298.15,
  temp0 =         298.15,
  ntt =           3,
  gamma_ln =      1.0,
  ig =            -1,
  ntp =           0,
  barostat =      0,
  nmropt =        0,
  pencut =        -1,
  ntb =           0,
  ntwprt =        0,
 /
