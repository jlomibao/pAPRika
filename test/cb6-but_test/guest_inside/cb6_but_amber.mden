L0  Nsteps           time(ps)         Etot             EKinetic        
L1  Temp             T_solute         T_solv           Pres_scal_solu  
L2  Pres_scal_solv   BoxX             BoxY             BoxZ            
L3  volume           pres_X           pres_Y           pres_Z          
L4  Pressure         EKCoM_x          EKCoM_y          EKCoM_z         
L5  EKComTot         VIRIAL_x         VIRIAL_y         VIRIAL_z        
L6  VIRIAL_tot       E_pot            E_vdw            E_el            
L7  E_hbon           E_bon            E_angle          E_dih           
L8  E_14vdw          E_14el           E_const          E_pol           
L9  AV_permMoment    AV_indMoment     AV_totMoment     Density          dV/dlambda      
L0      500  0.1000000000E+01  -.6737748145E+03  0.7905015649E+02
L1 0.2418227029E+03  0.2418227029E+03  -.7528249710E+09  0.1000000000E+01
L2 0.1000000000E+01  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L3 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L4 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L5 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L6 0.0000000000E+00  -.7528249710E+03  -.6649489397E+02  0.1323052646E+04
L7 -.1249654986E+03  0.2448264375E+02  0.1447265799E+03  0.6288481013E+02
L8 0.5361656168E+01  -.2121872914E+04  0.0000000000E+00  0.0000000000E+00
L9 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L0     1000  0.2000000000E+01  -.6606502381E+03  0.8357971746E+02
L1 0.2556791040E+03  0.2556791040E+03  -.7442299556E+09  0.1000000000E+01
L2 0.1000000000E+01  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L3 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L4 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L5 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
L6 0.0000000000E+00  -.7442299556E+03  -.6656644730E+02  0.1334801842E+04
L7 -.1303926711E+03  0.3165025217E+02  0.1472891237E+03  0.5865127404E+02
L8 0.6360082128E+01  -.2126023411E+04  0.0000000000E+00  0.0000000000E+00
L9 0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
