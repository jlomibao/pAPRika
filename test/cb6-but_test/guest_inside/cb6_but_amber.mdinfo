
 NSTEP =        0   TIME(PS) =       0.000  TEMP(K) =   347.46  PRESS =     0.0
 Etot   =      -700.2066  EKtot   =       113.5813  EPtot      =      -813.7879
 BOND   =        11.0080  ANGLE   =       109.4369  DIHED      =        52.9578
 1-4 NB =         5.8595  1-4 EEL =     -2127.9655  VDWAALS    =       -67.7555
 EELEC  =      1326.4920  EGB     =      -123.8211  RESTRAINT  =         0.0000
