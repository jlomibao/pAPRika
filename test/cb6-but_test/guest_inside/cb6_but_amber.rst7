default_name                                                                    
  125  0.2000000E+01
  -0.2862357   5.0210270   1.9762810   1.2167425   4.9357017   2.4264023
  -0.3844800   4.3270230   0.6891264  -0.9800746   4.3689881   3.0842247
  -0.5787558   6.0693692   1.8464876   1.7120820   5.9134892   2.4287662
   1.7366940   4.1261271   1.3483672   1.1276367   4.2015843   3.7006631
   3.1524684   3.8015769   1.2309007   2.2435184   3.9808553   4.5890160
  -0.2324488   3.9045134   4.0680056   0.8588709   3.7452974   0.3929423
  -0.6125350   3.3792624   5.0990260   1.0963116   3.1458432  -0.6104907
   3.3671960   3.6874085   0.1621195   3.7694895   4.6433059   1.5658819
   1.9955305   3.9809964   5.6566945   3.0029267   4.7454487   4.3886808
  -4.4747249   2.3341787   0.8127981  -3.5953643   3.6060404   1.0118312
  -3.7597467   1.6389601  -0.2900813  -4.4280669   1.6850174   2.1170518
  -5.5031851   2.6030134   0.5455292  -4.1102809   4.5539200   0.8173476
  -2.5352784   3.4752039   0.0149229  -3.2110858   3.5023014   2.4078786
  -1.5029672   4.5012878  -0.2504267  -2.4515380   4.4717823   3.1553345
  -3.6287300   2.3151748   2.9731900  -2.6459814   2.2892849  -0.7299617
  -3.4380541   1.9415864   4.1301439  -1.9763995   2.0428319  -1.6899212
  -1.2519406   4.4704082  -1.3169482  -1.8939165   5.4998971  -0.0237700
  -2.7457423   4.3314593   4.2018473  -2.7386588   5.4247823   2.6962180
  -3.9467055  -2.7232687   1.2819793  -4.6477665  -1.4152909   0.9888749
  -2.7905677  -2.7160916   0.3672241  -3.6480683  -2.6938859   2.6433408
  -4.6028724  -3.5808859   1.0939020  -5.6845944  -1.5444178   0.6575788
  -3.8186502  -0.7536567  -0.0399912  -4.6050859  -0.7467274   2.2605033
  -4.3428810   0.3918430  -0.8476038  -5.2945014   0.5498567   2.4390676
  -4.0814676  -1.6103289   3.2601894  -2.7471484  -1.5715104  -0.3704638
  -3.9811365  -1.3648866   4.4578760  -1.9561884  -1.4246128  -1.2531284
  -4.1465437   0.3030379  -1.9223136  -5.4128766   0.4323941  -0.6132962
  -5.5263119   0.5448717   3.5103648  -6.1910174   0.4409833   1.8179067
   0.2964558  -5.1804917   2.5347410  -1.1695673  -4.8601249   2.1845997
   1.1259226  -4.5905205   1.4634431   0.4015842  -4.4709568   3.8002174
   0.3502461  -6.2583692   2.7263707  -1.8179099  -5.7386470   2.0882356
  -0.9562118  -4.2359689   0.8286845  -1.6486683  -4.0584219   3.2320696
  -2.1519883  -4.0390963   0.0110059  -3.0955087  -3.9106723   3.4195633
  -0.7742288  -3.9295075   4.2953800   0.3651259  -4.0798888   0.4425790
  -0.9498966  -3.5131837   5.4442680   0.7616580  -3.5745612  -0.5884496
  -1.9664706  -4.0803447  -1.0684932  -2.9236187  -4.7639169   0.2950075
  -3.3815494  -3.7450456   4.4646401  -3.6311211  -4.7769391   3.0144258
   4.5160449  -2.3335553   3.5679127   3.6554383  -3.5804641   3.3331364
   4.5237235  -1.6873538   2.2430001   3.8212614  -1.6464089   4.6600490
   5.5385261  -2.5745394   3.8807973   4.0496700  -4.5707803   3.5886563
   3.2053736  -3.4652372   1.9672966   2.5925307  -3.3761449   4.3050193
   2.5674307  -4.6186224   1.3405403   1.6539458  -4.5083212   4.6049740
   2.7874210  -2.3217336   5.1324489   3.7885768  -2.4210370   1.3454408
   2.0418243  -2.0010637   6.0547861   3.7094524  -2.2035480   0.1803693
   2.8103441  -4.7099111   0.2756013   2.9288118  -5.5242949   1.8411361
   1.3865026  -4.4714896   5.6673091   2.2251062  -5.4077381   4.3475631
   4.0566415   2.7311619   3.2781496   4.6793685   1.3526196   3.5232422
   3.5763686   2.5787393   1.8762524   3.0474872   2.7392637   4.2991131
   4.7424934   3.5861244   3.2871828   5.7597053   1.3486578   3.7084246
   4.5769555   0.6075631   2.2631827   3.9394050   0.7633574   4.6763210
   5.3452012  -0.5380990   1.9727052   4.3974137  -0.4496953   5.2928969
   2.9942871   1.6443121   5.0784153   3.9896307   1.3746702   1.2831711
   2.2650708   1.5649405   6.0744289   3.8663235   1.1096026   0.0755146
   5.6092775  -0.5239547   0.9089859   6.2530174  -0.5486560   2.5868675
   4.1252573  -0.4531327   6.3546670   5.4830659  -0.5339129   5.1675894
   0.3988413  -0.0898150   0.8132177   1.4510038   0.2196822   0.8321188
  -0.1951969   0.8113642   0.6178095   0.3511029  -0.7782190  -0.0394362
  -0.0123385  -0.6917829   2.0935630  -1.0453690  -1.0165519   1.9187392
  -0.0998866   0.3562151   3.2543639   0.5958082  -1.5782091   2.3117278
   0.9082812   0.7760794   3.3568401  -0.6960860   1.2250894   2.9497295
  -0.6396719  -0.1347914   4.5941213  -0.0211324  -0.9798474   4.9204258
  -0.5800631   0.6241417   5.3838411  -1.7010751  -0.4056553   4.5371498
  -0.6845082   1.2159709  -4.6649129   0.4678926  -0.8192524  -5.6800729
  -2.0645854   2.8425420 -12.6586670
   0.3820484  -0.0904298   0.1507597   0.0025392   0.1109606  -0.2175377
   0.3521660  -0.1293411   0.1753423   0.1525692  -0.0781746  -0.1104855
   0.9943965   0.0704669   0.0036270  -0.1708340   0.1999607  -0.0137408
  -0.0215421  -0.0547252  -0.1577574  -0.0316495   0.3534242  -0.1346369
  -0.2803896  -0.3498469   0.0330385  -0.3108474  -0.2388961   0.1203511
  -0.1546877   0.0350578  -0.2127026   0.1340013  -0.1920752   0.1972611
  -0.0337865   0.0581268   0.0697974   0.0335587   0.2512432  -0.0221463
  -0.3103241  -0.2365486   0.0146548  -1.6201939   0.6995378   0.0409038
  -0.4712533  -1.0370549   0.0959034   0.2214225  -0.6292100   0.5819455
   0.0760194  -0.0350325  -0.6678910  -0.2403719   0.0188305  -0.1940323
   0.3129915  -0.0480310  -0.0777846  -0.0310919  -0.1346584  -0.0880471
  -0.3398454   0.4560977   1.1436313  -0.0512304   0.1791220   0.0724924
  -0.0589497  -0.0704320   0.2105309   0.0258379   0.2721598  -0.1129044
   0.0544437  -0.2023523  -0.1331803  -0.2713610  -0.1102026  -0.0721583
   0.1375816   0.1705878   0.0932147  -0.1196384  -0.5484609  -0.0176144
   0.1516392  -0.0938381   0.0663582  -0.2465188   0.1494757   0.0299592
  -0.6791102  -0.6270812  -0.3079146   0.0937980   0.0049261  -0.9189411
  -0.2530844   0.8850726   0.0862603   0.0895736  -0.1210873  -0.3292257
  -0.2238908   0.3921856   0.1527800   0.0672794   0.0254302  -0.3911791
   0.0001583  -0.4855279  -0.2302693  -0.4169784   0.1937424   0.1571378
   0.0634049   0.3800068  -0.9305914   0.3261715  -1.3444703  -0.7979792
  -0.1269318  -0.2938498   0.0188701   0.2837292   0.3902457   0.0331531
   0.2866069  -0.0642457  -0.4079025  -0.0349070   0.2919851  -0.1116930
  -0.0802737   0.0776136  -0.1919437  -0.1639581   0.2209912   0.1755631
  -0.1534276  -0.2147826   0.0017846   0.0059829  -0.0329065  -0.0377410
  -0.2923727   0.4999138  -0.5732461   0.1759654  -0.2286997  -0.8631745
   0.2769153  -0.6923670  -0.0283137  -0.4001191  -1.2207362   0.5848316
  -0.0247348  -0.0887299  -0.1615153   0.1957227   0.0414427   0.0441326
  -0.0327958  -0.2060439   0.3311503   0.2277738  -0.1089553  -0.0758476
  -0.4306009  -0.2636876  -0.9452703  -0.4852061   0.5203833   0.1110391
   0.0242636  -0.4105244  -0.0930257  -0.1237436  -0.1677364   0.0698089
  -0.1225914  -0.3094830   0.0578160   0.0264895   0.2053528   0.0545402
  -0.4430258   0.2599690   0.2453888  -0.0427858   0.3793899  -0.2127006
  -0.0046087  -0.1430844   0.0119575  -0.1367583   0.0211194   0.1117237
  -0.9858315  -0.1751561  -0.1106665   0.0003291  -0.2146790   0.6617516
  -0.0321751   0.3373515   0.0179941  -0.8771603   0.6196011   0.3102695
  -0.2253924  -0.0455996  -0.1500470   0.0958528   0.1232116   0.1163808
  -0.1074686   0.1722287   0.5418606  -0.0770772   0.1468624   0.0786053
  -0.1601690   0.4621882   0.0476726  -0.5750933  -0.2401081  -0.2018467
   0.0808946   0.0489834  -0.0488813   0.2491003  -0.2179942   0.0662190
  -0.1469188  -0.0104723  -0.0519727  -0.1141179   0.2178163  -0.0628446
   0.0973202  -0.2086230   0.0712493   0.2795262   0.0946545  -0.1243327
  -0.2162771  -0.0184255   0.0179147  -0.2543802  -0.1011516   0.0882637
   1.1631585  -0.2537062   0.2320503  -1.3493031  -0.0995666   0.7396470
   0.4593970  -0.7338473   0.1391067  -1.2385295  -0.6223560   0.2151722
   0.2288176   0.2475469  -0.0612970  -0.0229696   0.0015270  -0.1000523
  -0.0611392  -0.0764862   0.0987979   0.3891557  -0.1540284   0.3492506
   1.3104208  -0.5480626  -0.9618922   0.0520098   1.2015217  -0.3454371
   0.1844163  -0.0408926  -0.2046211  -0.1796640  -0.1350623   0.1386730
   0.2943324  -0.2385288   0.1240728  -0.0351638  -0.1629228  -0.1248070
   0.0375085  -0.3431282  -0.0477316  -0.2542756  -0.2588906  -0.2538090
  -0.2060997  -0.1637138   0.0914248   0.1521893  -0.2087829  -0.0908381
  -1.5376741   1.2814863  -0.4251653   0.0804533  -0.5794040   0.4431744
   0.8533259   0.1332555   0.1219758  -0.0715640   0.6989694  -1.4109406
  -0.1850183   0.0099715  -0.0172591   0.2971635  -1.4601497   1.0578175
   0.9679247   0.6835688  -0.6433898   0.9120545  -0.1220349  -0.0016043
  -0.0150321  -0.0849475   0.1239055   0.2615211  -1.0264574   0.1261902
  -0.4113390  -0.3552869  -0.0791440  -0.7489930  -0.6152810   0.0920363
  -0.4660086  -0.3918126   0.7446889  -0.1722135   0.4182674   1.4568208
   0.0072387   0.0337257  -0.0294669   0.3655409   0.2511509  -0.1339932
  -1.9111387  -0.3386885   0.5818328  -0.2086428   0.8436483  -0.1129246
  -0.1046650  -0.0085974   0.0570338  -0.0734341  -0.0025888   0.0514147
  -0.1156329   0.0562424  -0.0059352
