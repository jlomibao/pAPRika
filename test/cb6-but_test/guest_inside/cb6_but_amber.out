
          -------------------------------------------------------
          Amber 16 PMEMD                              2016
          -------------------------------------------------------

| PMEMD implementation of SANDER, Release 16

| Run on 02/19/2019 at 16:12:13

|   Executable path: pmemd
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/cb6-but_test/guest_inside
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|   MDIN: cb6_but_amber.in                                                      
|  MDOUT: cb6_but_amber.out                                                     
| INPCRD: cb6-but-dum.rst7                                                      
|   PARM: cb6-but-dum.prmtop                                                    
| RESTRT: cb6_but_amber.rst7                                                    
|   REFC: cb6_but_amber.inpcrd                                                  
|  MDVEL: mdvel                                                                 
|   MDEN: cb6_but_amber.mden                                                    
|  MDCRD: cb6_but_amber.nc                                                      
| MDINFO: cb6_but_amber.mdinfo                                                  
|  MDFRC: mdfrc                                                                 


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          0,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        0,                                                           
  ncyc =          0,                                                           
  dt =            0.002,                                                       
  nstlim =        1000,                                                        
  ntpr =          250,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          250,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      0,                                                           
  nmropt =        0,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
  ntwprt =        0,                                                           
 /                                                                             


Note: ig = -1. Setting random seed to   385675 based on wallclock time in 
      microseconds.
 
| Conditional Compilation Defines Used:
| PUBFFT
| BINTRAJ
| EMIL

| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| INFO:    Reading atomic numbers from topology file.
| Duplicated    0 dihedrals

| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   
--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       0, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     250, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     250, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Molecular dynamics:
     nstlim  =      1000, nscm    =      1000, nrespa  =         1
     t       =   0.00000, dt      =   0.00200, vlimit  =  20.00000

Langevin dynamics temperature regulation:
     ig      =  385675
     temp0   = 298.15000, tempi   = 298.15000, gamma_ln=   1.00000

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

| Intermolecular bonds treatment:
|     no_intermolecular_bonds =       1

| Energy averages sample interval:
|     ene_avg_sampling =     250

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps

 
 Number of triangulated 3-point waters found:        0
| Dynamic Memory, Types Used:
| Reals                3895
| Integers            14371

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------


 NSTEP =        0   TIME(PS) =       0.000  TEMP(K) =   347.46  PRESS =     0.0
 Etot   =      -700.2066  EKtot   =       113.5813  EPtot      =      -813.7879
 BOND   =        11.0080  ANGLE   =       109.4369  DIHED      =        52.9578
 1-4 NB =         5.8595  1-4 EEL =     -2127.9655  VDWAALS    =       -67.7555
 EELEC  =      1326.4920  EGB     =      -123.8211  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------


 NSTEP =      250   TIME(PS) =       0.500  TEMP(K) =   195.80  PRESS =     0.0
 Etot   =      -691.9243  EKtot   =        64.0057  EPtot      =      -755.9299
 BOND   =        20.0859  ANGLE   =       147.5025  DIHED      =        63.1226
 1-4 NB =         6.7906  1-4 EEL =     -2130.8931  VDWAALS    =       -65.4990
 EELEC  =      1334.9805  EGB     =      -132.0199  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------


 NSTEP =      500   TIME(PS) =       1.000  TEMP(K) =   241.82  PRESS =     0.0
 Etot   =      -673.7748  EKtot   =        79.0502  EPtot      =      -752.8250
 BOND   =        24.4826  ANGLE   =       144.7266  DIHED      =        62.8848
 1-4 NB =         5.3617  1-4 EEL =     -2121.8729  VDWAALS    =       -66.4949
 EELEC  =      1323.0526  EGB     =      -124.9655  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------


 NSTEP =      750   TIME(PS) =       1.500  TEMP(K) =   233.71  PRESS =     0.0
 Etot   =      -672.5343  EKtot   =        76.3988  EPtot      =      -748.9331
 BOND   =        18.2947  ANGLE   =       156.1809  DIHED      =        59.0634
 1-4 NB =         7.3499  1-4 EEL =     -2122.6970  VDWAALS    =       -65.4935
 EELEC  =      1324.8039  EGB     =      -126.4354  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------

| RE_POSITION Moving by   0.843676  0.285945  0.831347

 NSTEP =     1000   TIME(PS) =       2.000  TEMP(K) =   255.68  PRESS =     0.0
 Etot   =      -660.6502  EKtot   =        83.5797  EPtot      =      -744.2300
 BOND   =        31.6503  ANGLE   =       147.2891  DIHED      =        58.6513
 1-4 NB =         6.3601  1-4 EEL =     -2126.0234  VDWAALS    =       -66.5664
 EELEC  =      1334.8018  EGB     =      -130.3927  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------


      A V E R A G E S   O V E R       4 S T E P S


 NSTEP =     1000   TIME(PS) =       2.000  TEMP(K) =   231.75  PRESS =     0.0
 Etot   =      -674.7209  EKtot   =        75.7586  EPtot      =      -750.4795
 BOND   =        23.6284  ANGLE   =       148.9248  DIHED      =        60.9305
 1-4 NB =         6.4656  1-4 EEL =     -2125.3716  VDWAALS    =       -66.0135
 EELEC  =      1329.4097  EGB     =      -128.4534  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------


      R M S  F L U C T U A T I O N S


 NSTEP =     1000   TIME(PS) =       2.000  TEMP(K) =    22.19  PRESS =     0.0
 Etot   =        11.1760  EKtot   =         7.2551  EPtot      =         4.3777
 BOND   =         5.1497  ANGLE   =         4.3294  DIHED      =         2.0800
 1-4 NB =         0.7276  1-4 EEL =         3.5464  VDWAALS    =         0.5178
 EELEC  =         5.5167  EGB     =         2.8600  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|  NonSetup CPU Time in Major Routines:
|
|     Routine           Sec        %
|     ------------------------------
|     Nonbond           1.58   86.86
|     Bond              0.00    0.14
|     Angle             0.03    1.67
|     Dihedral          0.16    8.89
|     Shake             0.00    0.22
|     RunMD             0.04    2.18
|     Other             0.00    0.04
|     ------------------------------
|     Total             1.82

|  Generalized Born CPU Time:
|
|     Routine                 Sec        %
|     ------------------------------------
|     Radii Calc              0.38   20.77
|     Diagonal Calc           0.62   34.01
|     Off Diagonal Calc       0.57   31.18
|     GBSA LCPO Calc          0.00    0.03
|     ---------------------------------
|     Total                   1.56   85.96

|  Final Performance Info:
|     -----------------------------------------------------
|     Average timings for last    1000 steps:
|     Elapsed(s) =       1.82 Per Step(ms) =       1.82
|         ns/day =      95.06   seconds/ns =     908.86
|
|     Average timings for all steps:
|     Elapsed(s) =       1.82 Per Step(ms) =       1.82
|         ns/day =      95.06   seconds/ns =     908.86
|     -----------------------------------------------------

|  Setup CPU time:            0.00 seconds
|  NonSetup CPU time:         1.82 seconds
|  Total CPU time:            1.82 seconds     0.00 hours

|  Setup wall time:           0    seconds
|  NonSetup wall time:        2    seconds
|  Total wall time:           2    seconds     0.00 hours
